src/main/java/com/rcv/Tabulator.java

```diff
@@ -461,8 +461,7 @@ private CandidateStatus getCandidateStatus(String candidate) {
       eliminated.add(label);
       Logger.log(
           Level.INFO,
-          "Eliminated %s in round %d because it represents undeclared write-ins. It had "
-              + "%s votes.",
+          "Eliminated candidate \"%s\" in round %d because it represents undeclared write-ins. It had %s votes.",
```

Comment:
Here and elsewhere: I thought we were supposed to limit lines to 100 characters. Are you proposing changing it to a higher value?

Reply:
Very weird that IntelliJ isn't correcting this during file reformats (in fact, it looks like it's doing the opposite). I'm just going to go with it since it's easier to go with the automated recommendation than to fight it.

Reply:
Sigh. This seems like another case in which there's a mysterious inconsistency between your machine, Jon's, and mine in how IntelliJ is applying the style rules. For me, it complains whenever someone exceeds 100 characters.

Reply:
Ping pong :(
At least it doesn't affect the functionality.